src/phoenix/experimental/evals/templates/template.py

```diff
@@ -26,6 +26,16 @@ def __init__(self, text: str, delimiters: List[str] = [DEFAULT_START_DELIM, DEFA
         self._validate()
         self.variables = self._parse_variables()
 
+    def append(self, template: Union[str, "PromptTemplate"]) -> None:
+        """
+        Appends a template to the current template.
+        Args:
+            template (Union[str, PromptTemplate]): The template to append.
+        """
+        template = normalize_template(template)
+        self.text += template.text
```

Comment:
Suggestion: Use `sep.join` with a configure `sep` defaulting to `\n` for readabilty for formatted prompt.